lib/workers/repository/update/branch/auto-replace.ts

```diff
@@ -288,6 +288,7 @@ export async function doAutoReplace(
           newDigest,
         );
       } else if (
+        !currentDigest &&
         currentDigestShort &&
```

Comment:
i wonder when we don't have current digest but a short current digest 🤔

Reply:
It seems that this is possible with github actions: https://github.com/renovatebot/renovate/blob/bc7d0595d0c42f84669ceb7884bef3f54cd6bd42/lib/modules/manager/github-actions/extract.ts#L109-L112